### DISCUSSION TITLE
Difference between x64v1/v2/v3?

### DISCUSSION THREAD

**Discussion by NovaAndrom3da** [xanmod/kernel#344] [Q&A] [answered]
I wasn't able to find any information- what's the difference between the v1, v2, and v3 versions of the x86_64 kernel?

**Comment by monarc99** [comment, marked as answer]
https://en.wikipedia.org/wiki/X86-64#Microarchitecture_levels

**Reply by NovaAndrom3da** [reply]
Oh okay, thank you.

**Comment by figue** [comment]
You can use this official script to check yours:

https://dl.xanmod.org/check_x86-64_psabi.sh
